### ISSUE TITLE
XMLHttpRequest is not defined - Node

### ISSUE THREAD

**Issue by MikeCarbone** [kozakdenys/qr-code-styling#238]
How can I avoid this? Only triggered when I include an image. I'm adding the `jsdom` and `nodeCanvas` library too. Anything else I can do? This error gets triggered in both Next.js API and Express

```
const options = {
      width: 800,
      height: 800,
      data: `data123`,
      image: "https://image-url.png",
      dotsOptions: {
          color: "#212121",
          type: "rounded"
      },
      backgroundOptions: {
          color: "#ffffff",
      },
      imageOptions: {
          crossOrigin: "anonymous",
          margin: 20
      }
 }

  const qrCodeImage = new QRCodeStyling({
    jsdom: JSDOM,
    nodeCanvas, // this is required
    ...options,
    imageOptions: {
        saveAsBlob: true,
        crossOrigin: "anonymous",
        margin: 20
    }
  });
```

**Comment by kozakdenys** [comment]
Hi @MikeCarbone, thanks for opening this issue, I am looking into this 👀

**Comment by kozakdenys** [comment]
@MikeCarbone no luck for now, I will try to fix it tomorrow morning

**Comment by MikeCarbone** [comment]
Thank you for looking into this @kozakdenys ! Let me know if I can help with more details

**Comment by kozakdenys** [comment]
Hi @MikeCarbone the issue should be fixed in [v1.8.1](https://github.com/kozakdenys/qr-code-styling/releases/tag/v1.8.1)
Also, I have added an example to this [repo](https://github.com/kozakdenys/qr-code-styling-examples/blob/master/examples/nodejs/src/main.ts) 🎉
You can clone, run `npm i`, `npm run build`, and `npm start` to get generated QRs.
If any problem, please reopen this issue

**Comment by MikeCarbone** [comment]
Can confirm it's working now. Thank you for your quick work on this @kozakdenys!
